src/atom_query/data/fundraiser_atoms.json

```diff
@@ -215,7 +215,7 @@
   "3b7c0c51531ac108f84a8ff99a32d2beead82352": 452,
   "3bd79d993f6c069eeb5dfc82f3c6701f2a701cea": 6784,
   "3c672b4ef87c1d90a616f07314fe9dae9e9fe2f3": 4621.85,
-  "3c71b260f2b1b09e6a7f61741418f608a6c6b888": 1056379.42,
```

Comment:
was this done with a tool or?

Reply:
I did it by hand, it needs to be triple-checked.